Echo.json5

```diff
@@ -20,6 +20,7 @@
     { name: 'AROptionsLotConditionReport', value: true },
     { name: 'AROptionsEnableSales', value: true },
     { name: 'AREnableViewingRooms', value: true },
+    { name: 'AROptionsNewFirstInquiry', value: true },
```

Comment:
This is my first echo pr - What does `true` mean here? If that's what we want, ✅ .

Reply:
`true` in this context means that the feature is enabled.